integration-test/src/main/resources/application.yml

```diff
@@ -53,6 +53,9 @@ integrationtest:
     crn:
     name:
     workloadPassword: "Admin@123"
+  saasuser:
+    accesskey: Y3JuOmFsdHVzOmlhbTp1cy13ZXN0LTE6Y2xvdWRlcmFfc2Fhczp1c2VyOm1vY2t1c2VyQHVtcy5tb2Nr
```

Comment:
please remove this!

Reply:
nvm, I do not need different test user with recent changes, although I am not sure why should I remove this. there is already another user in application.yml with similar properties

Reply:
just fyi: this is not an AWS access/secret, but base64 encoded access/secret for CDP

Reply:
ok, accepted